generators/server/templates/src/main/resources/application.yml.ejs

```diff
@@ -40,17 +40,34 @@ micronaut:
         enabled: true
   security:
     enabled: true
+<%_ if (authenticationType === 'oauth2') { _%>
+    oauth2:
+      enabled: true
+      login-uri: /oauth2/authorization{/provider}
+      callback-uri: /login/oauth2/code{/provider}
```

Comment:
This placeholder looks funny. Is the `/` supposed to be inside the placeholder?

Reply:
Yes, that's how it is specified in the Micronaut docs.  https://micronaut-projects.github.io/micronaut-security/latest/guide/index.html#authorization-code